### DISCUSSION TITLE
End of use of PF (13) & JSF for me: TabView - Events and Validation

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [primefaces/community#196] [PrimeFaces] [answered]
Having upgraded WildFly 30.x with Mojarra 4.0.4 on PF 13.0 once again I tried the very basis "TabView - Events and Validation" example.

Still get:

`SEVERE [jakarta.enterprise.resource.webcontainer.faces.context] (default task-4) jakarta.el.PropertyNotFoundException: /index.xhtml @37,117 listener="#{tabbedView.onTabChange}": Target Unreachable, identifier 'tabbedView' resolved to null: jakarta.el.PropertyNotFoundException: /index.xhtml @37,117 listener="#{tabbedView.onTabChange}": Target Unreachable, identifier 'tabbedView' resolved to null`

With over 8 years of using PF & over 4 years trying to use "TabView" on Mojarra & WildFly enough is enough. It always used to work on RichFaces and PF 5.x/6.x

I'm migrated all my 'Entities' to GSON/JSON and are now using "Kotlin Multiplatform" for my UI for both Web, Android & iOS. 

Thanks for those who have supported me over the years & some of those terse responders too. Good-bye.

**Comment by tandraschko** [comment, marked as answer]
thanks for ignoring all discussion on the forum; thats not a PF problem, its a problem in your setup like stated like 10 times before.

**Reply by melloware** [reply]
Agreed I have proved over and over that this was not a PF issue with TabView.
